src/test/scala/com/fulcrumgenomics/ForLoopTesting.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics
+
+import com.fulcrumgenomics.Loopy._
+import com.fulcrumgenomics.testing.UnitSpec
+
+object Loopy {
+  @inline def for1(from: Int, to:Int)(f: Int => Unit): Unit = {
+    var i = from
+    while (i < to) {
+      f(i)
+      i += 1
+    }
+  }
+
+  @specialized def for2[T](from: T)(next: T => T)(check: T => Boolean)(f: T => Unit): Unit = {
+    var t = from
+    while (check(t)) {
+      f(t)
+      t = next(t)
+    }
+  }
+}
```

Comment:
Is this file meant to be part of the PR?  If so, why not use the functions in `FgBioDef`?